daemon/src/routes_maker.rs

```diff
@@ -95,15 +95,14 @@ pub async fn post_sell_order(
     cfd_actor_address: &State<Address<maker_cfd::Actor>>,
     _auth: Authenticated,
 ) -> Result<status::Accepted<()>, status::BadRequest<String>> {
-    let order = Order::from_default_with_price(order.price, Origin::Ours)
-        .map_err(|e| status::BadRequest(Some(e.to_string())))?
-        .with_min_quantity(order.min_quantity)
-        .with_max_quantity(order.max_quantity);
-
     cfd_actor_address
-        .do_send_async(maker_cfd::NewOrder(order))
+        .do_send_async(maker_cfd::NewOrder {
+            price: order.price,
+            min_quantity: order.min_quantity,
+            max_quantity: order.max_quantity,
+        })
         .await
-        .expect("actor to always be available");
+        .map_err(|e| status::BadRequest(Some(e.to_string())))?;
```

Comment:
I know you just took it over but I really think this should be a 500. This failing has nothing to do with the user's request.